apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -797,6 +797,10 @@ type GlobalConfig struct {
 	// +listType=set
 	Tags []string `json:"tags,omitempty"`
 
+	// OriginDetectionUnified enables Unified mechanism for Origin Detection
+	// +optional
+	OriginDetectionUnified *bool `json:"originDetectionUnified,omitempty"`
```

Comment:
We usually put in the name the suffix `enabled` like `OriginDetectionUnifiedEnabled` or we add a sub option `enabled` like
```yaml
global:
  originDetectionUnified:
    enabled: true
```

the second solution is when we think that we might add later other option linked to "originDetectionUnified".
If we are not sure the best is to use the second option.

Reply:
Done 👍 I did it using the second option just in case.